CHANGELOG.asciidoc

```diff
@@ -40,7 +40,7 @@ This release also includes changes from <<release-3-6-6, 3.6.6>> and <<release-3
 * Added new data type `DT` to represent periods of time.
 * Added Gherkin support for Date.
 * Extended `datetime()` function to produce a current server date.
-* Added list filtering functions `all` and `any`.
+* Added list filtering functions `all`, `any`, and `none`.
```

Comment:
there is a major deprecation to document here. there should also be provider upgrade docs as `none()` is an important part of server interactions.